common/src/main/java/com/wynntils/commands/FunctionCommand.java

```diff
@@ -47,17 +54,20 @@ public String getDescription() {
     @Override
     public LiteralArgumentBuilder<CommandSourceStack> getCommandBuilder() {
         return Commands.literal(getCommandName())
-                .then(Commands.literal("list").executes(this::listFunctions))
+                .then(Commands.literal("list")
+                        .then(Commands.argument("type", StringArgumentType.word())
```

Comment:
Maybe allow also just `/function list` as an alias of `/function list all`? You can do that either by handling the exception in the function on `context.getArgument` if there is no argument, or by calling `listFunction` with either "all" or `context.getArgument("type")` as argument, instead of using a direct method reference. I think we have used both methods about 50% of the time...